### PR TITLE
Fixed bug

### PR DESCRIPTION
**Pull request by Ta1al**
- Field value exceeding 1024 character limit.
- Able to view details of a specific flow

_I have tested this and it works as it should. I'd still recommend testing it just to be sure._ 

> Issue #48

### CODE REVIEW

File: src/commands/listflows.js [hunk 1]
```diff
@@ -1,43 +1,76 @@
...
+  aliases: [ "listflow", "flows", "flow" ],
   permissionRequired: 1, // 0 All, 1 Mods, 2 Admins, 3 Server Owner, 4 Bot Admin, 5 Bot Owner
-  checkArgs: (args) => !args.length
+  checkArgs: (args) => args.length <= 1
 };
 
 const { limitFlows, formatExplanation, generateTip } = require("../constants/index.js"), config = require("../../config.json");
 
-module.exports.run = async (message, _, gdb, { prefix }) => {
+module.exports.run = async (message, _, gdb, { prefix, content: query }) => {
```
Comment (promise) [inline src/commands/listflows.js]: Rather replace `_` with `[ query ]`, or perhaps `[ flowID ]` to make it easier to understand in the future.

File: src/commands/listflows.js [hunk 2]
```diff
@@ -1,43 +1,76 @@
...
+  query;
```
Comment (promise) [inline src/commands/listflows.js]: ?

File: src/commands/listflows.js [hunk 3]
```diff
@@ -1,43 +1,76 @@
...
+        },
+        fields: [
+          {
+            name: "**Triggers:**",
+            value: triggers.length > 1024 ? triggers.slice(0, 1002) + " [Too long to show...]" : triggers,
+          },
+          {
+            name: "**Actions:**",
+            value: actions.length > 1024 ? actions.slice(0, 1002) + " [Too long to show...]" : actions,
+          }
```
Comment (promise) [inline src/commands/listflows.js]: `name` property should not be bold.

**Review by promise — CHANGES_REQUESTED** [review]
I've requested some changes, but everything else looks good Talal!

File: src/commands/listflows.js (hunk 2 above)
Comment (Ta1al) [reply src/commands/listflows.js]: I just put that there cuz eslint was bugging me for `query` not being used when I working on something else. Forgot to remove it...

File: src/commands/listflows.js (hunk 1 above)
Comment (Ta1al) [reply src/commands/listflows.js]: Done.

File: src/commands/listflows.js (hunk 3 above)
Comment (Ta1al) [reply src/commands/listflows.js]: Done.

**Review by promise — APPROVED** [review]

**Pull request merged**